File: tests/indexing/parsing/lang/fixtures/go/NestedBlocks.go
package example

func process(xs []int, threshold int) int {
	total := 0
	for _, x := range xs {
		if x > threshold {
			switch {
			case x > 100:
				total = total + 100
				total = total - 1
			default:
				total = total + x
			}
		}
	}
	return total
}

File: tests/indexing/parsing/lang/fixtures/javascript/NestedBlocks.js
function scan(xs, threshold) {
  let total = 0;
  for (const x of xs) {
    if (x > threshold) {
      switch (x % 2) {
        case 0:
          total = total + 100;
          total = total - 1;
          break;
        default:
          total = total + x;
      }
    }
  }
  return total;
}
